crates/foundry/evm/core/src/backend/mod.rs

```diff
@@ -579,17 +585,21 @@ impl<
     > Backend<BlockT, TxT, EvmBuilderT, HaltReasonT, HardforkT, TransactionErrorT, ChainContextT>
 {
     /// Creates a new Backend with a spawned multi fork thread.
-    pub fn spawn(fork: Option<CreateFork<BlockT, TxT, HardforkT>>) -> Self {
-        Self::new(MultiFork::spawn(), fork)
+    pub fn spawn(
+        fork: Option<CreateFork<BlockT, TxT, HardforkT>>,
+        local_predeploys: impl IntoIterator<Item = Predeploy>,
```

Comment:
Aren't these mutually exclusive?

In that case, it would have stronger typing if defined as an `enum`